oat-sa/tao-item-runner-qti-fe
# fix/aut-657 Reorder to min number

Related to https://oat-sa.atlassian.net/browse/AUT-657

**Description:**
For GraphicOrderInteraction changed a starting order from the minimum available number, 
Bonus: fixed an issue when clicking on text label was not register an event and throw an error to console.

**Changes:**
* Changed active when unselecting shape.
* Add pointer-events none to label and remove unused code
* Added missed UT 

**How to check:**
* Create an item with GraphicOrderInteraction
* Add multiple shapes (5 is enough)
* Go to Responses 
* Select in order 1,2,3,4,5
* Deselect in order 1,5,2 
* See selection is 1 all the time (or minimum available)
* An extra check, clicking on the number itself will not throw an error anymore

**Requires:**
* Swap in package.json to `"@oat-sa/tao-item-runner-qti": "github:oat-sa/tao-item-runner-qti-fe#fix/AUT-657_reorder-to-min-number"`(fixed) for taoQTIItem extention.
* Rebundle `npx grunt taoqtiitembundle` from tao/views/build.
